api/core/listen_result.go

```diff
@@ -71,3 +72,15 @@ func isSubscribedTask(request *ListenResultRequest, e *execution.Execution) bool
 func isSubscribedOutput(request *ListenResultRequest, e *execution.Execution) bool {
 	return array.IncludedIn([]string{"", "*", e.Output}, request.OutputFilter)
 }
+
+func isSubscribedExecution(request *ListenResultRequest, e *execution.Execution) bool {
```

Comment:
I suggest to rename this function `isSubscribedExecution` to something that explicitly show the tags like: `isSubscribedExecutionTags`

Reply:
what about `isSubscribedToTags` at the same time I can rename the `isSubscribedOutput` to `isSubscribedToOutput`

Reply:
I'm ok with your suggestion. I just like more context to the names.